web/src/scripts/containers/ComponentProps.jsx

```diff
@@ -145,7 +145,24 @@ class ComponentProps extends Component {
     if (!prop) return
 
     const {styles} = this.props
-    const {name} = prop
+    const {name, type} = prop
+
+    if (type === 'object') {
```

Comment:
if we're rendering an object prop, we don't get to see the property name? probably need to do something different here.